src/isc-dhcp/Makefile

```diff
@@ -2,21 +2,20 @@
 SHELL = /bin/bash
 .SHELLFLAGS += -e
 
-MAIN_TARGET = isc-dhcp-relay_$(ISC_DHCP_VERSION)_$(CONFIGURED_ARCH).deb
-DERIVED_TARGETS = isc-dhcp-relay-dbgsym_$(ISC_DHCP_VERSION)_$(CONFIGURED_ARCH).deb
+MAIN_TARGET = $(ISC_DHCP_RELAY)
+DERIVED_TARGETS = $(ISC_DHCP_RELAY_DBG)
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# Remove any stale files
-	rm -rf ./isc-dhcp
+	rm -rf ./isc-dhcp-$(ISC_DHCP_VERSION)
 
 	# Clone isc-dhcp repo
-	git clone https://salsa.debian.org/dhcp-team/isc-dhcp.git
-	pushd ./isc-dhcp
+	dget -u http://deb.debian.org/debian/pool/main/i/isc-dhcp/isc-dhcp_$(ISC_DHCP_VERSION_FULL).dsc
```

Comment:
Why did you decide to change the method of obtaining the source code?

Reply:
I initially picked it up from snmpd/Makefile.

I checked using the existing method and I see some patches missing when using the tag debian/4.4.1-2. The CVE related patches are available when using the dget method.

Reply:
Which CVE-related patches are not available using the existing method? I see the three that I am familiar with are still there at the 4.4.1-2 tag: https://salsa.debian.org/dhcp-team/isc-dhcp/-/tree/debian/4.4.1-2/debian/patches

Reply:
We dont need to have a copy of the patch. The code comes patched if we use the dget method.

Reply:
We don't need to have a copy of the patch using the old method, either. The patches are now in the repo, as you can see from my link above.

Reply:
Thanks. Reverted to the old method.